dsl/pact.go

```diff
@@ -412,9 +412,16 @@ func (p *Pact) VerifyProvider(t *testing.T, request types.VerifyRequest) (types.
 	for _, example := range res.Examples {
 		t.Run(example.Description, func(st *testing.T) {
 			st.Log(example.FullDescription)
+
 			if example.Status != "passed" {
-				t.Errorf("%s\n%s\n", example.FullDescription, example.Exception.Message)
+				if strings.Contains(example.FullDescription, "[PENDING]") {
+					t.Logf("NOTICE: This interaction is in a pending state because it has not yet been successfully verified by %s. If this verification fails, it will not cause the overall build to fail. Read more at https://pact.io/pending", p.Provider)
```

Comment:
This message is not strictly true, as the tags are taken in to consideration. If you want a per interaction message, I'll populate the pending_message for you.

Reply:
Yes I know, I just needed something in for the folks to start using today. It's good enough for our initial use case. This was the least amount of change/impact to this code base in order to get the functionality.

As soon as the JSON property is in place I'll switch to it.